src/utilities/new_versions.jl

```diff
@@ -0,0 +1,122 @@
+body_info(::KeepEntry, name::AbstractString) = "This keeps the compat entries for earlier versions.\n\n"
+body_info(::DropEntry, name::AbstractString) = "This drops the compat entries for earlier versions.\n\n"
+body_info(::NewEntry, name::AbstractString) = "This is a brand new compat entry. Previously, you did not have a compat entry for the `$(name)` package.\n\n"
+
+title_parenthetical(::KeepEntry) = " (keep existing compat)"
+title_parenthetical(::DropEntry) = " (drop existing compat)"
+title_parenthetical(::NewEntry) = ""
+
+function new_compat_entry(
+    ::KeepEntry,
+    old_compat::AbstractString,
+    new_compat::AbstractString,
+)
+    return "$(strip(old_compat)), $(strip(new_compat))"
+end
+
+function new_compat_entry(
+    ::Union{DropEntry, NewEntry},
+    old_compat::AbstractString,
+    new_compat::AbstractString,
+)
+    return "$(strip(new_compat))"
+end
+
+function compat_version_number(ver::VersionNumber)
+    (ver.major > 0) && return "$(ver.major)"
+    (ver.minor > 0) && return "0.$(ver.minor)"
+
+    return "0.0.$(ver.patch)"
+end
+
+function subdir_string(subdir::AbstractString)
+    if !isempty(subdir)
+        subdir_string = " for package $(splitpath(subdir)[end])"
+    else
+        subdir_string = ""
+    end
+end
+
+function pr_info(
+    version_verbatim::Nothing,
+    name::AbstractString,
+    compat_entry_for_latest_version::AbstractString,
+    compat_entry::AbstractString,
+    subdir_string::AbstractString,
+    pr_body_keep_or_drop::AbstractString,
+    pr_title_parenthetical::AbstractString,
+)
+    new_pr_title = string(
+        "CompatHelper: add new compat entry for ",
```

Comment:
Dunno if we want to add another dep just for this, but https://github.com/invenia/MultilineStrings.jl might be nice to make these strings a little nicer.

Reply:
Looks like MultilineStrings.jl has no dependencies, right?

I'd be fine adding it, since it's so lightweight. These multi-line strings really are annoying to deal with.

Reply:
My only issue w/ MLS is that VSCode doesn't understand these variables are being used inside of the macro :/

![Screen Shot 2021-06-08 at 9 43 42 AM](https://user-images.githubusercontent.com/4356074/121206378-073e8f00-c83e-11eb-9c9c-4b58efe9f517.png)

Reply:
I don't feel strongly either way.